### DISCUSSION TITLE
google-generativeai requires google-api-core and grpcio

### DISCUSSION THREAD

**Discussion by khalidelayadi** [flet-dev/flet#4295] [Packages] [answered]
cannot download google-api-core and grpcio while building apk

**Comment by FeodorFitsner** [comment, marked as answer]
grpcio is available now for Flet 0.25 pre-release: https://github.com/flet-dev/flet/discussions/3157

**Comment by khalidelayadi** [comment]
yes thank you available on flet_desktop==0.25.0.dev3721
